code/components/citizen-server-impl/include/state/SyncTrees_Five.h

```diff
@@ -1023,7 +1023,74 @@ struct CDoorScriptInfoDataNode { bool Parse(SyncParseState& state) { return true
 struct CDoorScriptGameStateDataNode { bool Parse(SyncParseState& state) { return true; } };
 struct CHeliHealthDataNode { bool Parse(SyncParseState& state) { return true; } };
 struct CHeliControlDataNode { bool Parse(SyncParseState& state) { return true; } };
-struct CObjectCreationDataNode { bool Parse(SyncParseState& state) { return true; } };
+
+struct CObjectCreationDataNode {
+	uint32_t m_model;
+
+	bool Parse(SyncParseState& state)
+	{
+		int unk0 = state.buffer.Read<int>(5);
```

Comment:
"Created By", probably a subsystem ID. If it's 0 or 2, it's a dummy object (not sure where the game uses these - in GTA3 this was for, say, map objects that fell over, but the player moved away).